guided-trials-default-custom-user-data/default-custom-user-data.gradle

```diff
@@ -135,6 +135,17 @@ void addCiMetadata() {
             buildScan.value 'CI agent ID', agentId
         }
     }
+
+    if (isGitHubActions()) {
+        def repo = System.getenv('GITHUB_REPOSITORY')
+        def runId = System.getenv('GITHUB_RUN_ID')
+        if (repo && runId) {
+            buildScan.link 'CI job', "https://github.com/$repo/actions/runs/$runId"
+        }
+        if (System.getenv('GITHUB_HEAD_REF')) {
+            buildScan.tag 'Fork'
```

Comment:
should we add a custom value with the 'value' of 'GITHUB_HEAD_REF', which is the 'The branch of the head repository.' ?

Reply:
Sure, we can do that. What would we call the key?

Reply:
Forked from ?

Reply:
I've decided to remove it for now as I don't see a real use case.